buildSrc/src/main/groovy/io/vantiq/extsrc/assy/tasks/AssemblyResourceGeneration.groovy

```diff
@@ -501,7 +515,16 @@ class AssemblyResourceGeneration extends DefaultTask {
             log.debug("\t processing property {}: {}", pName, pDesc)
 
             // Properties are required if there is no default...
-            Map<String, Object> propDesc = [ required: pDesc.default == null ]
```

Comment:
simplistic setting of required (above) was incorrect. Use `required` property as provided (this may be new).